### PR TITLE
Dr 24 12 raphael

### PR DESCRIPTION
**Pull request by thea-m**
https://github.com/BetaMasaheft/Documentation/issues/2719

### CODE REVIEW

File: 2001-3000/LIT2387Taamme.xml [hunk 1]
```diff
@@ -33,17 +35,40 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+         <textClass>
+            <keywords>
+               <term key="ChristianLiterature"/>
+               <term key="Hagiography"/>
+               <term key="Miracle"/>
+            </keywords>
+         </textClass>
+         <langUsage><language ident="en">English</language> <language ident="gez">Gǝʿǝz</language></langUsage>
+         <abstract>
+            <p>This records refers to a collection of more than one miracle of the archangel Raphael. The miracles may circulate in varying constitution as a collection by themselves,
```
Comment (nafisa-valieva) [inline 2001-3000/LIT2387Taamme.xml:47]: record

**Review by nafisa-valieva — APPROVED** [review]
Very nice and helpful! Thank you!

File: new/LIT7163MRKidnapping.xml [hunk 2]
```diff
@@ -0,0 +1,75 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7163MRKidnapping" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Miracle of Raphael: On a boy kidnapped by a wild animal</title>
```
Comment (CarstenHoffmannMarburg) [inline new/LIT7163MRKidnapping.xml:8]: The text tells us, that it was an elephant, who kidnapped the boy.

**Review by CarstenHoffmannMarburg — APPROVED** [review]

**Comment by thea-m** [comment]
Thank you!

**Review by CarstenHoffmannMarburg — APPROVED** [review]

**Pull request merged**